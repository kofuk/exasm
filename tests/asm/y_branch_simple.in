beqz r3, 0x7d
bnez r7, 0x7d
bmi r3, 0x7d
bpl r7, 0x7d
j 0x7D
nop
