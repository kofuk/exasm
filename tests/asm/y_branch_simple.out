@00 10000011 01111101 // beqz r3, 0x7D
@02 10001111 01111101 // bnez r7, 0x7D
@04 10010011 01111101 // bmi r3, 0x7D
@06 10011111 01111101 // bpl r7, 0x7D
@08 11000000 01111101 // j 0x7D
@0a 00000000 00000000 // nop
